### PR TITLE
[TextTranslation] Move Text Translation back under cognitive services directory

### PR DESCRIPTION
**Pull request by catalinaperalta**
I noticed Text Translation was checked in outside of `cognitiveservices/`, this PR moves Text Translation files back under the `cognitiveservices/` directory.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json
Comment (catalinaperalta) [inline specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json:1]: @weshaggard I created a new directory "TextTranslation" to mirror the name of the directory being added at the top level of the cognitiveservices/ directory. However, the previous swaggers exist under: https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane/TranslatorText

How should we proceed here? Add the new directory or move the files under the TranslatorText/ directory?

Comment (catalinaperalta) [reply specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json:1]: I believe the correct name for the service is TextTranslation, @MikeyMCZ can correct me if not

Comment (weshaggard) [reply specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json:1]: That is a good question. At least for now I'd suggest we use the name we would like to have. The only potential issue I can think of is the openapi diff between this version and the older version might not connect in the tooling as they would appear to be different services. We might need to do diff breaking changes comparisons manually to ensure this new version doesn't break from the older one.

Comment (catalinaperalta) [reply specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json:1]: Just catching up on this PR again, another idea would be to first create a PR with the original swagger checked in the new destination folder and then merge this PR after that one so we get the tool to check for us. Would that work to make sure we do the breaking changes diff?

Comment (weshaggard) [reply specification/cognitiveservices/data-plane/TextTranslation/stable/v3.0/openapi.json:1]: I don't think that will help us much here. The issue that I think might be a problem is fact that if we move the swagger to a new folder the tooling wouldn't be able to find the older version of the swagger to do the comparisons with. So we might need to manually run those comparisons if we want to make sure we didn't break anything.

**Comment by catalinaperalta** [comment]
Closing as this PR is old and there are new discussions about where the ai specs should go

**Pull request closed without merging**